lib/get-files.js

```diff
@@ -1,23 +1,6 @@
-import { readFileSync } from 'fs';
-import walkSync from 'walk-sync';
-import { join } from 'path';
+import { globbySync } from 'globby';
 
 export default function getFiles(cwd, providedGlob) {
-  let ignoreFile;
-
-  try {
-    ignoreFile = readFileSync(join(cwd, '.gitignore'), 'utf8')
```

Comment:
we seem to be missing the ignore file parsing that we used to have 🤔 can't we just append these ignores to the globby ignore?

Reply:
Globby does this for us, via `ignoreFiles: ['**/.gitignore', '**/.eslintignore']`. It even handles nested one!

Reply:
Also supporting the `.eslintignore` was a big reason for this change.